### ISSUE TITLE
Benchmark with a good dataset for absolute intensity quantification

### ISSUE THREAD

**Issue by ypriverol** [bigbio/ibaqpy#42]
We should reanalyze some good datasets for absolute intensity base quantification datasets where the expression/concentrations are known to benchmark the package and the algorithms behind. 

@daichengxin @WangHong007 can you let me know a dataset that we can use to benchmark IBAQ methods.

**Comment by ypriverol** [comment]
Potential datasets: 

- [ ] PXD014765, PXD012836:  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8788469/. @WangHong007 They both need to be annotated. 
- [ ] PXD011725: https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/full/10.1002/pmic.202000093. @WangHong007 need to be annotated.

**Comment by WangHong007** [comment]
@ypriverol PXD014765 and PXD012836 link to the same thesis, and I've completed anontation of the former, please check it, especially the bio_replicate column. For the later, it's similar to the former and it included three organisms of yeast, but I found limited experimental design info in PRIDE record and no sign in the thesis. Should we guess it?

**Comment by ypriverol** [comment]
@daichengxin can you help here?

**Comment by daichengxin** [comment]
You can see the informations in Supplementary file. Combined mode of samples: standard (T2), high temperature (T3), osmotic shock (T4), low pH (T5), anaerobic (T6) and three (technical) replicates for each condition (A, B, C). Plus 6 bulk samples (Bulk, A-F)

**Comment by WangHong007** [comment]
@daichengxin Yes but this is experimental info of PXD014765, I don't know how to generate SDRF of PXD012836 since they both linked to the same thesis.
